projects/collection-editor-library/src/lib/components/library/library.component.ts

```diff
@@ -50,6 +52,8 @@ export class LibraryComponent implements OnInit, AfterViewInit {
       this.fetchContentList();
       this.telemetryService.telemetryPageId = this.pageId;
       this.childNodes = _.get(this.collectionhierarcyData, 'childNodes');
+    }, err => {
+      this.toasterService.error('Something went wrong');
```

Comment:
@rajnish-dargan are not we using resource bundles here?

Reply:
No, currently we using one file for the lables, added this in that file.